internal/ingress/controller/nginx.go

```diff
@@ -668,6 +672,11 @@ Error: %v
 //
 //nolint:gocritic // the cfg shouldn't be changed, and shouldn't be mutated by other processes while being rendered.
 func (n *NGINXController) OnUpdate(ingressCfg ingress.Configuration) error {
+	concurrentlyReloadWorkers := n.store.GetBackendConfiguration().ConcurrentlyReloadWorkers
+	if !concurrentlyReloadWorkers && n.workersReloading {
+		return errors.New("worker reload already in progress, requeuing reload")
```

Comment:
should this be an error, or just a warning?

Reply:
The idea of returning an error is to pick up on the existing behaviour on the queue, which will re-queue in case of error here https://github.com/kubernetes/ingress-nginx/blob/8227888ba0a9b09afb4514284ef89f331d4c60b9/internal/task/queue.go#L130